src/cc/libbpf.h

```diff
@@ -24,6 +24,11 @@
 extern "C" {
 #endif
 
+enum bpf_probe_attach_type {
+	probe_entry,
```

Comment:
please make it upper case. Like: BPF_PROBE_ENTRY
to reduce the chance of conflicts.